x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/AbstractFunctionTestCase.java

```diff
@@ -1071,28 +1071,33 @@ public void trackSignature() {
     }
 
     @AfterClass
-    public static void renderTypesTable() throws IOException {
+    public static void renderDocs() throws IOException {
         if (System.getProperty("generateDocs") == null) {
             return;
         }
         String name = functionName();
         if (binaryOperator(name) != null) {
-            renderTypesTable(List.of("lhs", "rhs"));
+            renderTypes(List.of("lhs", "rhs"));
             return;
         }
         if (unaryOperator(name) != null) {
-            renderTypesTable(List.of("v"));
+            renderTypes(List.of("v"));
             return;
         }
         FunctionDefinition definition = definition(name);
         if (definition != null) {
-            renderTypesTable(EsqlFunctionRegistry.description(definition).argNames());
+            EsqlFunctionRegistry.FunctionDescription description = EsqlFunctionRegistry.description(definition);
+            renderTypes(description.argNames());
+            renderParametersList(description.argNames(), description.argDescriptions());
+            renderDescription(description.description());
+            boolean hasExamples = renderExamples(EsqlFunctionRegistry.functionInfo(definition));
+            renderFullLayout(name, hasExamples);
             return;
```

Comment:
I intentionally only plugged this into the functions because I wanted to be able to think about operators in a separate PR.

Reply:
But I wrote all the functions so we can use them for operators if we want. Just plug them in the same was as we do above.